### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by Alexendoo** [ccrama/Slide#82]
## Stacktrace
```java.lang.RuntimeException: Unable to start activity ComponentInfo{me.ccrama.redditslide/me.ccrama.redditslide.Activities.SubredditOverview}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.support.v4.widget.DrawerLayout.setDrawerLockMode(int, int)' on a null object reference
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2325)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2387)
at android.app.ActivityThread.access$800(ActivityThread.java:151)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1303)
at android.os.Handler.disp
atchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:135)
at android.app.ActivityThread.main(ActivityThread.java:5254)
at java.lang.reflect.Method.invoke(N
ative Method)
at java.lang.reflect.Method.invoke(Method.java:372)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.support.v4.widget.DrawerLayout.setDrawerLockMode(int, int)' on a null object reference
at me.ccrama.redditslide.Activities.SubredditOverview.doSubSidebar(SubredditOverview.java:635)
at me.ccrama.redditslide.Activities.SubredditOverview.setD
ataSet(SubredditOverview.java:1157)
at me.ccrama.redditslide.Activities.SubredditOverview.onCre
ate(SubredditOverview.java:187)
at android.app.Activity.performCre
ate(Activity.java:5990)
at android.app.Instrument
ation.callActivityOnCre
ate(Instrument
ation.java:1106)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2278)... 10 more```
Reported via Slide Crash Capture

**Comment by SolarAquarion** [comment]
I also got this thing

**Comment by ccrama** [comment]
Fixed with 3a198b29f1598e917a20223352d633d27d0d9797
